msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/ui/DiscussionTopicBean.java

```diff
@@ -556,6 +558,62 @@ public void setPostFirst(String postFirst)
       topic.setPostFirst(Boolean.valueOf(false));
     }
   }
+
+  /**
+   * @return Returns the postAnonymous status.
+   */
+  public String getPostAnonymous()
+  {
+    LOG.debug("getPostAnonymous()");
+    if ("".equals(postAnonymous))
+    {
+      boolean isAnonymous = topic != null && topic.getPostAnonymous() != null && topic.getPostAnonymous().booleanValue();
+      postAnonymous = Boolean.valueOf(isAnonymous).toString();
+    }
+    return postAnonymous;
+  }
+
+  /**
+   * @param postAnonymous
+   * Set the postAnonymous status
+   */
+  public void setPostAnonymous(String postAnonymous)
+  {
+    if (LOG.isDebugEnabled())
```

Comment:
I don't know, I think best practice is to call isDebugEnabled() first because it is a cheap call and debug() isn't.